packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -906,12 +909,8 @@ export class BasicCrawler<
 }
 
 /** @internal */
-export async function basicCrawlerEnqueueLinks(options: BasicCrawlerEnqueueLinksOptions, requestQueue?: RequestQueue) {
-    return enqueueLinks({
-        // FIXME this does not make much sense, as the instance would be ignored by any crawler - the argument needs to be required
-        requestQueue: requestQueue ?? await RequestQueue.open(),
-        ...options,
-    });
+export async function basicCrawlerEnqueueLinks(options: EnqueueLinksOptions) {
```

Comment:
let's remove this and use `enqueueLinks` directly

Reply:
For basic crawlers only? I agree, but for the rest I think they should stay separate (+ makes testing easier)

Reply:
i would inline the usage of `basicCrawlerEnqueueLinks`, rest can stay the same (although i would like to not export those directly, but we can deal with that later, there are more candidates that could live in some `internals.js` barrel that would be listed in package exports, not won't be documented)